src/index.js

```diff
@@ -79,7 +79,8 @@ class Daemon extends EventEmitter {
     if (headless == null) {
       return cb(new Error('Could not load "headless" module'))
     }
-    headless((err, child, display) => {
+    var opts = {args: ['+extension', 'RANDR']}
```

Comment:
This thread shows that we should be doing `-extension` to remove the RANDR extension: http://stackoverflow.com/questions/17944234/xlib-extension-randr-missing-on-display-21-trying-to-run-headless-googl